docs/maui/get-started.md

```diff
@@ -144,6 +144,33 @@ builder
 
 To use the features of the toolkit please refer to the documentation pages for each specific feature.
 
+### [CommunityToolkit.Maui.CameraView](#tab/CommunityToolkitMauiCameraView)
+
+This package enables you to connect to a camera, display a preview from the camera and take photos.
+
+**Package name:** `CommunityToolkit.Maui.CameraView`
```

Comment:
```suggestion
**Package name:** `CommunityToolkit.Maui.Camera`
```

I've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.